### ISSUE TITLE
--recover-password ?p returns non-printable result

### ISSUE THREAD

**Issue by A-Kazak** [kimci86/bkcrack#91]
Greetings!

```
bkcrack --keys 79ef8f53 1a48082d c122db9b --recover-password 20 ?p
bkcrack 1.5.0 - 2022-07-07
[16:14:01] Recovering password
length 0-6...
length 7...
length 8...
length 9...
length 10...
length 11...
length 12...
length 13...
length 14...
8.0 % (724 / 9025)
[02:13:34] Password
as bytes: 27 42 71 37 28 6b 20 58 34 fc e8 ff ba 06
as text: 'Bq7(k X4ⁿΦ ║♠
```

  What was my mistake?
  How do I search for a password consisting of printable characters only?

Thank you.

**Comment by kimci86** [comment]
Hello, this is a known issue: #76.
There is no workaround at the moment. I will try to solve it soon.

**Comment by A-Kazak** [comment]
I hope that won't take too long :)
Anyway, I am ready to test a new version.

**Comment by kimci86** [comment]
The fix has just been released in version 1.6.0.

Example:
```
bkcrack --keys 79ef8f53 1a48082d c122db9b --recover-password 20 ?p
bkcrack 1.6.0 - 2024-01-02
[15:28:50] Recovering password
length 14...
Password: 'Bq7(k X4���� (as bytes: 27 42 71 37 28 6b 20 58 34 fc e8 ff ba 06)
Some characters are not in the expected charset. Continuing.
7.7 % (699 / 9025)
```
